lib/rack_middleware/custom_cookie_renamer.rb

```diff
@@ -0,0 +1,14 @@
+class CustomCookieRenamer
+  def initialize(app)
+    @app = app
+  end
+
+  def call(env)
+    if env["HTTP_COOKIE"] && !(env["HTTP_COOKIE"] =~ /\b#{APP_CONFIG.session_key}/)
+      env["HTTP_COOKIE"].sub!(/\b#{APP_CONFIG.old_session_key}=/, "#{APP_CONFIG.session_key}=")
```

Comment:
Should this check the case if old_session_key was being used? I guess the sub! does nothing if the cookie doesn't match old_session_key. Is that ok behavior?